server/app/lib/actions/fusor/deployment/cloud_forms/update_admin_password.rb

```diff
@@ -43,8 +43,8 @@ def run
               @io.close unless @io.closed?
 
             rescue Exception => e
+              @io.close if @io && !@io.closed?
               fail _("Failed to update admin password on appliance. Error message: #{e.message}")
-              @io.close unless @io.closed?
```

Comment:
Ok I can see why we're doing this check, especially in an exception block. I'm ok with it.